qemu/tests/win_virtio_driver_update_test.py

```diff
@@ -23,11 +26,19 @@ def run(test, params, env):
     :param params: Dictionary with the test parameters
     :param env: Dictionary with test environment
     """
-    cdrom_virtio_downgrade = params.get("cdrom_virtio_downgrade")
-
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
 
+    def change_virtio_media(cdrom_virtio):
+        """
+        change iso for virtio-win
+        :param cdrom_virtio: iso file
+        """
+        virtio_iso = utils_misc.get_path(data_dir.get_data_dir(),
+                                         cdrom_virtio)
+        logging.info("Iso is: %s" % virtio_iso)
```

Comment:
Should be `logging.info("Changing virtio iso image to '%s'" % virtio_iso)`

Reply:
Modified, pls have a check.
Thanks.